src/main/java/org/example/Order.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+public class Order {
+
+    private String orderName;
+    private int quantity;
+    private int price;
+
+    public Order() {
```

Comment:
You could skip adding empty constructor as it's added by default inside java compiler, when there are no other constructors